ComponentTextKit/TextKit/CKTextKitAttributes.h

```diff
@@ -104,6 +109,7 @@ struct CKTextKitAttributes {
     && maximumNumberOfLines == other.maximumNumberOfLines
     && shadowOpacity == other.shadowOpacity
     && shadowRadius == other.shadowRadius
+    && layoutManagerFactory == other.layoutManagerFactory
```

Comment:
Ok - can do that in a moment. Would you like me to squash the branch while I'm at it?